<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>displaySweep</class>
 <widget class="QWidget" name="displaySweep">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>468</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>5</horstretch>
    <verstretch>5</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="displayVLayout">
     <item>
      <widget class="MPLQTFig" name="MPLFig" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MPLQTNav" name="MPLNav" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="simplePlotTab">
        <attribute name="title">
         <string>Simple plo&amp;t</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QPushButton" name="startButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>&amp;Start</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="prev10Button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>-10</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="prevButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>&amp;Prev</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="sweepNumLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>S&amp;weep num:</string>
           </property>
           <property name="buddy">
            <cstring>NumSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="NumSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="nextButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>&amp;Next</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="next10Button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>+10</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="endButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>&amp;End</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="multiplotTab">
        <attribute name="title">
         <string>&amp;Multiplot</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="numSweepHLayout">
           <item>
            <widget class="QLabel" name="numSweepLabel">
             <property name="text">
              <string># of &amp;sweeps:</string>
             </property>
             <property name="buddy">
              <cstring>numSweepSpinBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="numSweepSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="rangeOptionHLayout">
           <item>
            <widget class="QPushButton" name="rangePushButton">
             <property name="text">
              <string>Ran&amp;ge</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="optionsPushButton">
             <property name="text">
              <string>Op&amp;tions</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="subMeanCheckBox">
           <property name="text">
            <string>s&amp;ubstract mean</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="randomCheckBox">
           <property name="toolTip">
            <string>If checked, select randomly the sweeps in sweep_list
Otherwise take the firsts</string>
           </property>
           <property name="text">
            <string>&amp;random</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="xjitterCheckBox">
           <property name="text">
            <string>x &amp;jitter</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="plotPushHLayout">
           <item>
            <widget class="QPushButton" name="replotPushButton">
             <property name="toolTip">
              <string>plot the same sweeps as previous call of Plot!</string>
             </property>
             <property name="text">
              <string>r&amp;eplot</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pltPushButton">
             <property name="text">
              <string>&amp;Plot!</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaVBoxLayout">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>302</width>
          <height>100</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="labelHLayout">
           <item>
            <widget class="QLabel" name="chanLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Chan</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="showLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Show</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="manLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Man</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="autoLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Auto</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="fullLabell">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Full</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="headerLine">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="chanVBoxLayout"/>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>x scale:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="xscaleCheckBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Auto</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="xUnitLabel">
            <property name="text">
             <string>&amp;x units:</string>
            </property>
            <property name="buddy">
             <cstring>xUnitComboBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="xUnitComboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="xminLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;x min:</string>
            </property>
            <property name="buddy">
             <cstring>xminDoubleSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="xminDoubleSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="locale">
             <locale language="English" country="UnitedKingdom"/>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="xmaxLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>x &amp;Max:</string>
            </property>
            <property name="buddy">
             <cstring>xmaxDoubleSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="xmaxDoubleSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="singleStep">
             <double>0.001000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPLQTFig</class>
   <extends>QWidget</extends>
   <header>neuropype.gui.MPLQTclasses</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MPLQTNav</class>
   <extends>QWidget</extends>
   <header>neuropype.gui.MPLQTclasses</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>prev10Button</tabstop>
  <tabstop>prevButton</tabstop>
  <tabstop>NumSpinBox</tabstop>
  <tabstop>nextButton</tabstop>
  <tabstop>next10Button</tabstop>
  <tabstop>endButton</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>xscaleCheckBox</tabstop>
  <tabstop>xminDoubleSpinBox</tabstop>
  <tabstop>xmaxDoubleSpinBox</tabstop>
  <tabstop>numSweepSpinBox</tabstop>
  <tabstop>rangePushButton</tabstop>
  <tabstop>optionsPushButton</tabstop>
  <tabstop>subMeanCheckBox</tabstop>
  <tabstop>randomCheckBox</tabstop>
  <tabstop>xjitterCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
